manual/python/index.md

```diff
@@ -66,9 +66,95 @@ caption="Example of graphics generated with PyROOT."
 
 ## Loading user libraries & jitting
 
-*Content is coming soon!*
+PyROOT allows to use *any C++ library* from Python, not only ROOT. This is possible thanks to the automatic and dynamic bindings between Python and C++ that PyROOT provides: without any prior generation of wrappers, at execution time, PyROOT can load and call into C++ code. The following sub-sections cover different supported scenarios for dynamic C++ code invocation.
+
+### Jitting strings with code
+
+ROOT comes with a C++ interpreter to which we can request to process a given piece of C++ code. Sometimes, if such code is short (e.g. the definition of a small function or class) or for rapid exploration/debugging, it is enough the place the C++ code in a Python string and give it to the interpreter. The code will be just-in-time compiled (jitted) and immediately available for invocation, as shown in the example below.
+
+```python
+import ROOT
+
+# Write some C++ code in a string
+cpp_code = """
+// Function definition
+int f(int i) { return i*i; }
+
+// Class definition
+class A {
+public:
+    A() { cout << "Hello PyROOT!" << endl; }
+};
+"""
+
+# Inject the code in the ROOT interpreter
+ROOT.gInterpreter.ProcessLine(cpp_code)
+
+# We find all the C++ entities in Python, right away!
+a = ROOT.A()   # this prints Hello PyROOT!
+x = ROOT.f(3)  # x = 9
+```
+
+### Including a header
+
+If the C++ code we want to use is in a header, we can also ask the interpreter to include it. Let's assume we have a header called `my_header.h` with the following content:
+
+```cpp
+int f(int i) { return i*i; }
 
-## IPython: running Python from C++
+class A {
+public:
+    A() { cout << "Hello PyROOT!" << endl; }
+};
+```
+
+We can execute the code below:
```

Comment:
That's a duplication, just say: We can include a header with code `bla` and then we can run exactly the same example as above, e.g., `ROOT.f(3)`.

Reply:
Ok, I read through all of it. You probably want the repetition to make very clear that you get the same outcome in different ways of loading the impls. So, fine by me to keep it as it is!